core/jobs/src/main/java/com/cognifide/aet/job/common/datafilters/removeregexp/ReplaceRegexpDataModifier.java

```diff
@@ -68,13 +74,14 @@ public String modifyPattern(String data) throws ProcessingException {
 
   @Override
   public String getInfo() {
-    return NAME + " DataModifier with parameters: " + DATA_REGEXP + ": " + dataRegExp + " " + PATTERN_REGEXP + ": " + patternRanges;
+    return NAME + " DataModifier with parameters: " + DATA_REGEXP + ": " + dataRegExp + " " + PATTERN_REGEXP + ": "
+        + patternRanges;
   }
 
-  private String modify(String data, String regexp) throws ProcessingException {
+  private String modify(String data, Pattern pattern) throws ProcessingException {
     String result = null;
     try {
-      result = data.replaceAll(regexp, StringUtils.EMPTY);
+      result = pattern.matcher(data).replaceAll(replacement);
```

Comment:
If user won't pass `value` argument, then `replacement` is `null`. Is that ok in this case?